apps/web/pages/auth/login.tsx

```diff
@@ -94,17 +103,22 @@ export default function Login({
     </Button>
   );
 
+  const communityLogin = queryParams.sso !== undefined && queryParams.sig !== undefined;
+
   const onSubmit = async (values: LoginValues) => {
     setErrorMessage(null);
     telemetry.event(telemetryEventTypes.login, collectPageParameters());
     const res = await signIn<"credentials">("credentials", {
       ...values,
-      callbackUrl,
-      redirect: false,
+      callbackUrl: communityLogin
+        ? `${COMMUNITY_URL}?sso=${queryParams.sso}&sig=${queryParams.sig}`
+        : queryParams.callbackUrl,
+      redirect: communityLogin,
```

Comment:
If we are dealing with a community login, then we need to enable redirection and provide the callbackUrl to let the sign in process know we are dealing with that community login.